RNAseq.nf

```diff
@@ -84,6 +84,11 @@ log.info "Working dir  : $workDir"
 log.info "Output dir   : ${params.out}"
 log.info "===================================="
 
+//Setup R libary directory
+params.rlocation = "/home/richam/R/nxtflow_libs/"
```

Comment:
Example [here](http://www.nextflow.io/docs/latest/config.html#scope-env) - should be able to do something like:
```python
params.rlocation = "$HOME/R/nxtflow_libs/"
```